### PR TITLE
XW-159 | add npm tasks

### PR DESCRIPTION
**Pull request by tomasznapieralski**
https://wikia-inc.atlassian.net/browse/XW-159

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,10 +6,16 @@
   "scripts": {
     "dev": "rm -rf www && gulp --nosync",
     "local": "rm -rf www && gulp",
-    "start": "rm -rf www && npm run build && node www/server/server.js",
-    "test": "rm -rf www && gulp build --type prod --nosync --testing && gulp karma && gulp node-test",
-    "coverage": "rm -rf www && gulp build --nosync --testing && gulp karma && gulp node-test",
-    "build": "gulp build --type prod --nosync"
+    "start": "npm run build && node www/server/server.js",
+
+    "tests": "npm run build-test && npm run test-front && npm run test-server",
```
Comment (rogatty) [inline package.json]: tests -> test

File: package.json [hunk 2]
```diff
@@ -6,10 +6,16 @@
   "scripts": {
     "dev": "rm -rf www && gulp --nosync",
     "local": "rm -rf www && gulp",
-    "start": "rm -rf www && npm run build && node www/server/server.js",
-    "test": "rm -rf www && gulp build --type prod --nosync --testing && gulp karma && gulp node-test",
-    "coverage": "rm -rf www && gulp build --nosync --testing && gulp karma && gulp node-test",
-    "build": "gulp build --type prod --nosync"
+    "start": "npm run build && node www/server/server.js",
```
Comment (rogatty) [inline package.json]: Why did you remove `rm -rf www`?

Comment (tomasznapieralski) [inline package.json]: i moved it to build step

Comment (rogatty) [inline package.json]: Oh ok, makes sense

**Comment by rogatty** [comment]
LGTM

**Comment by hakubo** [comment]
lgtm
